float8_experimental/float8_dynamic_linear.py

```diff
@@ -6,14 +6,25 @@
 """
 A wrapper around a `torch.nn.Linear` module which does fp8 compute.
 """
+
+import logging
+from typing import Any, Optional, Tuple
+
+import float8_experimental.config as config
+
 import torch
+import torch.nn as nn
+import torch.utils._pytree as pytree
 
 from float8_experimental.float8_tensor import (
     Float8Tensor,
     tensor_already_casted_to_fp8,
     to_fp8_no_autograd,
 )
 from float8_experimental.float8_utils import tensor_to_scale
+from torch._prims_common import suggest_memory_format
+
+log = logging.getLogger(__name__)
```

Comment:
https://docs.python.org/3/howto/logging.html#configuring-logging-for-a-library

I think you wanna register a null_handler

Reply:
removed this `log = ...` since I was using it only for debugging